kybra/compiler/kybra_generate/src/generators/ic_object/functions/clear_timer.rs

```diff
@@ -0,0 +1,15 @@
+use quote::quote;
+
+pub fn generate_clear_timer() -> proc_macro2::TokenStream {
+    quote! {
+        #[pymethod]
+        fn _kybra_clear_timer(
+            &self,
+            timer_id_py_object_ref: PyObjectRef,
+            vm: &VirtualMachine
+        ) -> PyObjectRef {
+            let timer_id: ic_cdk::timer::TimerId = timer_id_py_object_ref.try_from_vm_value(vm).unwrap();
+            ic_cdk::timer::clear_timer(timer_id).try_into_vm_value(vm).unwrap()
```

Comment:
Why don't we have to delete the callback from the global hashmap? I haven't seen the rest of the code yet so I might find out

Reply:
You know this now, but to document, we didn't end up needing to use hashmaps so we don't have to remove them :)